static/js/compose.js

```diff
@@ -724,13 +724,49 @@ function validate_stream_message() {
     return true;
 }
 
+//The function will check whether the recipient is in settings.CROSS_REALM_BOT_EMAILS or settings.ADMIN_DOMAIN
 function validate_private_message() {
     if (exports.recipient() === "") {
         compose_error("Please specify at least one recipient", $("#private_message_recipient"));
         return false;
     }
+    else {
+        var private_recipients = exports.recipient().split(/\s*[,;]\s*/);
+        var invalid_recipients = "";
+        var invalid_recipients_count = 0;
+        _.each(private_recipients, function (email) {
+            //This case occurs when exports.recipient() ends with ','
+            if(email === "") {
+                return;
+            }
+            if(page_params.cross_realm_bot_emails.indexOf(email) !== -1) {
+                return;
+            }
+            var index = email.length - page_params.domain.length;
+            //The email should contain '@' and atleast one char before '@'
+            if((index <= 1) || (email[index - 1] !== '@') || (email.substring(index) !== page_params.domain)) {
+                if(invalid_recipients_count === 0) {
+                    invalid_recipients = email;
+                }
+                else {
```

Comment:
else statements should be on the same line as `}` (I also added a lint rule for this!)